src/components/ThemeProvider/__stories__/ThemeProvider.stories.mdx

```diff
@@ -0,0 +1,79 @@
+import ThemeProvider from "../ThemeProvider";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createStoryMetaSettingsDecorator } from "../../../storybook";
+import { ThemeProviderTemplateOverview, ThemeProviderTemplateTwoButtons } from "./ThemeProviderTemplateTwoButtons";
+import { SystemTheme, Color } from "../ThemeProviderConstants";
+
+export const metaSettings = createStoryMetaSettingsDecorator({
+  component: ThemeProvider,
+  enumPropNamesArray: [], // List enum props here
+  iconPropNamesArray: [], // List props that are typed as icons here
+  actionPropsArray: [] // List the component's actions here
+});
+
+<Meta
+  title="Components/ThemeProvider"
+  component={ThemeProvider}
+  argTypes={metaSettings.argTypes}
+  decorators={metaSettings.decorators}
+/>
+
+# ThemeProvider
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+
+Please write here the component description
+
+<Canvas>
+  <Story
+    name="Overview"
+    args={{
+      theme: {
+        name: "overview-theme",
+        colors: {
+          [SystemTheme.LIGHT]: {
+            [Color.primaryColor]: "green"
+          },
+          [SystemTheme.DARK]: {
+            [Color.primaryColor]: "red"
+          },
+          black: {
+            "primary-color": "orange"
+          },
+          hacker: {
+            "primary-color": "yellow"
```

Comment:
Showing that both versions are available: enum with autocomplete or just a value. It's type safe to use both